hypothesis-python/src/hypothesis/strategies/_internal/core.py

```diff
@@ -1799,10 +1799,10 @@ def complex_numbers(
     check_valid_magnitude(min_magnitude, "min_magnitude")
     check_valid_magnitude(max_magnitude, "max_magnitude")
     check_valid_interval(min_magnitude, max_magnitude, "min_magnitude", "max_magnitude")
+    if min_magnitude is None:
+        min_magnitude = 0
```

Comment:
is there a way it should be noted that this is a deprecated path to flag for removal in the next major release?

Reply:
We don't need to note it here, because we have the `since` argument to note_deprecation. So when we turn that into an error in Hypothesis 6.0, this branch will not be covered by tests and we'll know to remove it along with the test.